src/main/scala/io/sdkman/changelogs/HadoopMigrations.scala

```diff
@@ -0,0 +1,70 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "059")
```

Comment:
This should be the next highest in order.
You can do this by running the following inside the project:
```
grep -R --no-filename 'ChangeLog(' | sort
```

Reply:
@ChethanUK lots of incoming traffic atm, can you make it `52` for me, please?

Reply:
Hi Sorry, was bit busy. Didn't check this github acc much these days. I just realized I had deleted this repo's fork.
Created NEW PR: https://github.com/sdkman/sdkman-db-migrations/pull/524